pydotorg/urls.py

```diff
@@ -1,71 +1,72 @@
-from django.conf.urls import include, url, handler404
```

Comment:
Looks good to me! Many many thanks for your help @berkerpeksag. Looking forward to seeing this deployed on python.org :-)